pkg/security/podsecuritypolicy/util/util.go

```diff
@@ -182,8 +183,9 @@ func PSPAllowsHostVolumePath(psp *extensions.PodSecurityPolicy, hostPath string)
 		return true
 	}
 
+	path := filepath.Clean(hostPath)
```

Comment:
no cleaning the path... there is no guarantee that the server is running the same OS as the node